lib/app.jsx

```diff
@@ -114,7 +114,12 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 
 	onAppCommand: function( command ) {
 		if ( command != null && typeof command === 'object' && command.action != null && this.props.actions.hasOwnProperty( command.action ) ) {
-			this.props.actions[ command.action ]( command );
+			// Special case for newNote since it needs the note bucket prop from this class
```

Comment:
oh okay, that makes sense. I wish some of this could have been a little less-custom in the way we have our app state, but at least we're in a much better spot than the notes client :smile:

maybe we could clarify in the comment, something like
> newNote expects a bucket to be passed in, but the action method itself wouldn't do that